app/controllers/facility_accounts_controller.rb

```diff
@@ -177,9 +185,6 @@ def show_statement
       when 'list'
```

Comment:
Minor: now that we've gotten rid of `recent`, we can probably just make this into a plain if/else condition rather than a case.